lib/widgets/body/how_to_play/how_to_play_title.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+import 'package:web/utils/styles.dart';
+
+class HowToPlayTitle extends StatelessWidget {
```

Comment:
For keeping consistency, I think it is better to separate files. For example, later if there is a widget with more than 10 components and we decide to keep it as one file it might be hard to find a specific component.